backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/CustomTypeMapper.kt

```diff
@@ -50,6 +55,8 @@ internal object CustomTypeMappers {
         result.associateBy { it.mappedClassId }
     }
 
+    internal val functionTypeMappersArityLimit = 33 // not including, i.e. [0..33)
```

Comment:
We might find useful to make this limit lower, to reduce the size of the binaries compiled with cache.
The following command can be used to check the size:
```
./gradlew clean dist hello1 -Ptest_with_cache_kind=static && du test.output/local/hello1/macos_x64/hello1.kexe
```
Please check if reducing the limit can help significantly with binary size.
It will be harder to reduce this limit in incremental releases later.

Reply:
```
functionTypeMappersArityLimit =  9: 714450 bytes
functionTypeMappersArityLimit = 33: 730834 bytes
```
Not too much

Reply:
What exactly are these numbers? My binary is about 1.5Mb.

Reply:
Sorry, corrected:
```
 9:	1421340
17:	1462484
33:	1569380
```